LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Modified BSD License
```

Comment:
The "instant approval" process I was following for committing to the other repo required a license file. (https://g3doc.corp.google.com/company/teams/opensource/patching/index.md?cl=head#instant) I'm not sure if we should have it here. Probably since it's a public repo we should have a license; and since modified BSD is pre-approved by Google it's a reasonable choice.